css.md

```diff
@@ -137,3 +138,19 @@ $_componentNameColor: #000;
 ```scss
 $_componentNameColor: $color1;
 ```
+
+### [2.3](#scss--comments): Comments
```

Comment:
This is missing the anchor tag above it like the other sections. The anchor tag is what the TOC links to.